### PR TITLE
RTL breadcrumbs

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Refactors breadcrumbs to use flexbox for layout.
- Adds RTL version of breadcrumbs for arabic pages.

## How to test this PR

1. Visit pages with long breadcrumbs in LTR and RTL languages, and resize to mobile and see that the breadcrumbs wrap correctly.

http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/
http://localhost:8000/language/ar/coronavirus/mortgage-and-housing-assistance/help-for-homeowners/

## Screenshots

Before:
Screen Shot 2023-06-02 at 5 22 21 PM

After:
Screen Shot 2023-06-02 at 5 23 49 PM

Arabic example:
Screen Shot 2023-06-06 at 10 50 18 AM

### CODE REVIEW

File: cfgov/agreements/jinja2/agreements/search.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
 <main class="content content__2-1 content__bleedbar">
     <div class="content_wrapper">
         <nav class="breadcrumbs" aria-label="Breadcrumbs">
-            <a href="/" class="breadcrumbs_link">Home</a>
-            <a href="/credit-cards/" class="breadcrumbs_link">Credit cards</a>
-            <a href="/credit-cards/agreements" class="breadcrumbs_link">Credit card agreement database</a>
+            <a href="/">Home</a>
+            <a href="/credit-cards/">Credit cards</a>
+            <a href="/credit-cards/agreements">Credit card agreement database</a>
```
Comment (chosak) [inline cfgov/agreements/jinja2/agreements/search.html:14]:
```suggestion
            <a href="/credit-cards/agreements/">Credit card agreement database</a>
```

**Review by chosak — APPROVED** [review]
Couple minor things, LG!

**Review by wpears — CHANGES_REQUESTED** [review]
The templates that implement their own breadcrumbs will need logic adding in their own slashes just like in breadcrumbs.html.

Currently, no slash is rendered (because it used to be added via pseudo element):
Screen Shot 2023-06-06 at 11 33 59 AM

File: cfgov/agreements/jinja2/agreements/archive.html [hunk 2]
```diff
@@ -8,9 +8,9 @@
 <main class="content content__2-1 content__bleedbar">
     <div class="content_wrapper">
         <nav class="breadcrumbs" aria-label="Breadcrumbs">
-            <a href="/" class="breadcrumbs_link">Home</a>
-            <a href="/data-research/credit-card-data/" class="breadcrumbs_link">Credit cards</a>
-            <a href="/credit-cards/agreements/" class="breadcrumbs_link">Agreement database</a>
+            <a href="/">Home</a>
+            <a href="/data-research/credit-card-data/">Credit cards</a>
+            <a href="/credit-cards/agreements/">Agreement database</a>
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/agreements/jinja2/agreements/archive.html]: Fixed in https://github.com/cfpb/consumerfinance.gov/pull/7790/commits/a950eb99a0b7a99c334c6f1d9aac67cd2e875806

**Comment by anselmbradford** [comment]
> I think it would be even better to extend the molecule such that these templates don't need their own custom logic, but can pass custom breadcrumbs to a macro, etc.

Added breadcrumbs template to agreements in https://github.com/cfpb/consumerfinance.gov/pull/7790/commits/a950eb99a0b7a99c334c6f1d9aac67cd2e875806

**Review by wpears — APPROVED** [review]
:cool:

**Pull request merged**